datasets/clinc_oos/README.md

```diff
@@ -52,34 +52,200 @@ pretty_name: CLINC150
+| 45 	| routing 	|
+| 46 	| meal_suggestion 	|
+| 47 	| tire_change 	|
+| 48 	| todo_list 	|
+| 49 	| card_declined 	|
+| 50 	| rewards_balance 	|
+| 51 	| change_accent 	|
+| 52 	| vaccines 	|
+| 53 	| reminder_update 	|
+| 54 	| food_last 	|
+| 55 	| change_ai_name 	|
+| 56 	| bill_due 	|
+| 57 	| who_do_you_work_for 	|
+| 58 	| share_location 	|
+| 59 	| international_visa 	|
+| 60 	| calendar 	|
+| 61 	| translate 	|
+| 62 	| carry_on 	|
+| 63 	| book_flight 	|
+| 64 	| insurance_change 	|
+| 65 	| todo_list_update 	|
+| 66 	| timezone 	|
+| 67 	| cancel_reservation 	|
+| 68 	| transactions 	|
+| 69 	| credit_score 	|
+| 70 	| report_fraud 	|
+| 71 	| spending_history 	|
+| 72 	| directions 	|
+| 73 	| spelling 	|
+| 74 	| insurance 	|
+| 75 	| what_is_your_name 	|
+| 76 	| reminder 	|
+| 77 	| where_are_you_from 	|
+| 78 	| distance 	|
+| 79 	| payday 	|
+| 80 	| flight_status 	|
+| 81 	| find_phone 	|
+| 82 	| greeting 	|
+| 83 	| alarm 	|
+| 84 	| order_status 	|
+| 85 	| confirm_reservation 	|
+| 86 	| cook_time 	|
+| 87 	| damaged_card 	|
+| 88 	| reset_settings 	|
+| 89 	| pin_change 	|
+| 90 	| replacement_card_duration 	|
+| 91 	| new_card 	|
+| 92 	| roll_dice 	|
+| 93 	| income 	|
+| 94 	| taxes 	|
+| 95 	| date 	|
+| 96 	| who_made_you 	|
+| 97 	| pto_request 	|
+| 98 	| tire_pressure 	|
+| 99 	| how_old_are_you 	|
+| 100 	| rollover_401k 	|
+| 101 	| pto_request_status 	|
+| 102 	| how_busy 	|
+| 103 	| application_status 	|
+| 104 	| recipe 	|
+| 105 	| calendar_update 	|
+| 106 	| play_music 	|
+| 107 	| yes 	|
+| 108 	| direct_deposit 	|
+| 109 	| credit_limit_change 	|
+| 110 	| gas 	|
+| 111 	| pay_bill 	|
+| 112 	| ingredients_list 	|
+| 113 	| lost_luggage 	|
+| 114 	| goodbye 	|
+| 115 	| what_can_i_ask_you 	|
+| 116 	| book_hotel 	|
+| 117 	| are_you_a_bot 	|
+| 118 	| next_song 	|
+| 119 	| change_speed 	|
+| 120 	| plug_type 	|
+| 121 	| maybe 	|
+| 122 	| w2 	|
+| 123 	| oil_change_when 	|
+| 124 	| thank_you 	|
+| 125 	| shopping_list_update 	|
+| 126 	| pto_balance 	|
+| 127 	| order_checks 	|
+| 128 	| travel_alert 	|
+| 129 	| fun_fact 	|
+| 130 	| sync_device 	|
+| 131 	| schedule_maintenance 	|
+| 132 	| apr 	|
+| 133 	| transfer 	|
+| 134 	| ingredient_substitution 	|
+| 135 	| calories 	|
+| 136 	| current_location 	|
+| 137 	| international_fees 	|
+| 138 	| calculator 	|
+| 139 	| definition 	|
+| 140 	| next_holiday 	|
+| 141 	| update_playlist 	|
+| 142 	| mpg 	|
+| 143 	| min_payment 	|
+| 144 	| change_user_name 	|
+| 145 	| restaurant_suggestion 	|
+| 146 	| travel_notification 	|
+| 147 	| cancel 	|
+| 148 	| pto_used 	|
+| 149 	| travel_suggestion 	|
+| 150 	| change_volume 	|
 
 ### Data Splits
 
-[More Information Needed]
+- Small : Small, in which there are only 50 training queries per each in-scope intent
+- Imbalanced : Imbalanced, in which intents have either 25, 50, 75, or 100 training queries.
+- Plus : OOS+, in which there are 250 out-of-scope training examples, rather than 100.
```

Comment:
```suggestion
The dataset comes in different subsets:

- `small` : Small, in which there are only 50 training queries per each in-scope intent
- `imbalanced` : Imbalanced, in which intents have either 25, 50, 75, or 100 training queries.
-`plus` : OOS+, in which there are 250 out-of-scope training examples, rather than 100.


|   name   |train|validation|test|
|----------|----:|---------:|---:|
|small|7600|     3100|  5500 |
|imbalanced|10625|     3100|   5500|
|plus|15250|     3100|   5500|

```